data/transit/route/train.json

```diff
@@ -393,12 +393,16 @@
       }
     },
     {
-      "displayName": "Dublin Commuter",
+      "displayName": "Commuter",
       "id": "dublincommuter-73b98a",
       "locationSet": {"include": ["ie"]},
       "tags": {
-        "network": "Dublin Commuter",
-        "operator": "Dublin Commuter",
+        "network": "Commuter",
```

Comment:
If the `network=Dublin Commuter` is already in widespread use, we probably shouldn't encourage retagging it to `network=Commuter`.  But other than that, the rest of the PR looks good 👍

Reply:
OK. Unfortunately I don't know how to edit the pr to fix the issue...

Reply:
> OK. Unfortunately I don't know how to edit the pr to fix the issue...

thats ok, I'll merge and put it back